### PR TITLE
EUTHEME-225: Implement grid component

### PR DESCRIPTION
**Pull request by Fefaine**

### CODE REVIEW

File: templates/views/views-view-grid.tpl.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * @file
+ * Default simple view template to display a rows in a grid.
+ *
+ * - $rows contains a nested array of rows. Each row contains an array of
+ *   columns.
+ *
+ * @ingroup views_templates
+ */
+?>
+<?php require(drupal_get_path('theme', 'europa') .'/templates/grid/grid.tpl.php'); ?>
```
Comment (drupol) [inline templates/views/views-view-grid.tpl.php]: I think that we should override the 'views-view-grid' theme definition properly. Meaning that we should use a preprocess for it and build a render array using the europa grid in it.

File: templates/table/table.component.inc [hunk 2]
```diff
@@ -37,6 +37,7 @@ function europa_preprocess_table(&$variables, $hook) {
   // We needs this patch in order to have the <tfoot> tag working properly:
   // https://www.drupal.org/node/806982#comment-12035531
   $variables['attributes']['class'][] = 'table-responsive';
+
```
Comment (drupol) [inline templates/table/table.component.inc]: Extra empty line.

File: templates/views-view/views-view.component.inc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+/**
+ * @file
+ * Contains component file.
+ */
+/**
+ * Implements hook_preprocess_views_view_grid().
+ */
+function europa_preprocess_views_view_grid(&$variables) {
+  $variables['theme_hook_suggestions'][] =  'grid';
```
Comment (drupol) [inline templates/views-view/views-view.component.inc]: I think that if we do that, it won't go in the grid preprocess. To be checked.

Comment (Fefaine) [inline templates/views-view/views-view.component.inc]: ho yes we do :)

Comment (Fefaine) [inline templates/views-view/views-view.component.inc]: Yes sorry i misunderstood your question, so what do you suggest to not duplicate the tpl?

Comment (Fefaine) [inline templates/views-view/views-view.component.inc]: OK i will merge, and we'll see later if we have a better solution

**Pull request merged**